### PR TITLE
Ability to save the clinician Expression in mainform

### PR DESCRIPTION
**Pull request by GowsalyaSG**

### CODE REVIEW

File: src/app/autorun/autorun.component.ts [hunk 1]
```diff
@@ -57,8 +57,7 @@ export class AutorunComponent implements OnInit {
   jobDetails: any;
 
   createCliniciansData(data: any) {
-    for (let i=0; i < this.clinicianModel.length; i++)
-    {
+    for (let i=0; i < this.clinicianModel.length; i++){
```
Comment (naveenkumarkandasamy) [inline src/app/autorun/autorun.component.ts:60]: @GowsalyaSG Indentation not good

**Comment by naveenkumarkandasamy** [comment]
@GowsalyaSG - What is the contect of the story? Did we miss something earlier and doing it now?

**Comment by aparnaacco879** [comment]
> @GowsalyaSG - What is the contect of the story? Did we miss something earlier and doing it now?

**Comment by aparnaacco879** [comment]
@naveenkumarkandasamy these changes are related to expressions in UI which was pending before.

File: src/app/form/form.component.ts [hunk 2]
```diff
@@ -155,23 +160,41 @@ export class MainFormComponent implements OnInit {
...
+      this.inputFormat = "Provide Online";
+      this.physicianMinCount = this.model[0].minCount;
+      this.physicianMaxCount = this.model[0].maxCount;
+      this.appMinCount = this.model[1].minCount;
+      this.appMaxCount = this.model[1].maxCount;
+      this.scribeMinCount = this.model[2].minCount;
+      this.scribeMaxCount = this.model[2].maxCount;
+      this.preferredOption = "utilization";
+      this.clinicianData = ['physician', 'app', 'scribe'];
+      for (var i = 0; i < this.model.length; i++) {
```
Comment (naveenkumarkandasamy) [inline src/app/form/form.component.ts:180]: Indentation

**Review by naveenkumarkandasamy — CHANGES_REQUESTED** [review]

**Pull request closed without merging**